lombok/src/main/java/com/baeldung/lombok/getter/GetterbooleanSameAccessor.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.lombok.getter;
+
+
+import lombok.Getter;
+
+/**
+ * Related Article Sections:
+ * 3.2. Two boolean Fields With the Same Accessor Name
+ *
+ */
+public class GetterbooleanSameAccessor {
+
+    @Getter
+    public boolean running = true;

```

Comment:
Maybe make these package-private, e.g. "boolean running" instead of "public boolean running"